src/Context/Swoole/src/SwooleContextHandler.php

```diff
@@ -41,7 +41,7 @@ public function switchToActiveCoroutine(): void
     public function splitOffChildCoroutines(): void
     {
         $pcid = Coroutine::getCid();
-        foreach (Coroutine::listCoroutines() as $cid) {
+        foreach (Coroutine::list() as $cid) {
```

Comment:
Do we need to take BC into account here?
```suggestion
        foreach (method_exists(Coroutine::class, 'list') ? Coroutine::list() : Coroutine::listCoroutines() as $cid) {
```